src/routes/index.js

```diff
@@ -2,10 +2,11 @@ const express = require("express");
 
 const router = express.Router();
 
+const verifyToken = require("./middleware/verifyToken");
 const userRoute = require("./user.route");
 const projectRoute = require("./project.route");
 
 router.use("/users", userRoute);
-router.use("/projects", projectRoute);
+router.use("/projects", verifyToken, projectRoute);
```

Comment:
여기에서 토큰검증을 하게되면, 저희 서비스 클라이언트에서 보내는 api 요청들도 이 라우터로 들어갈텐데 그러면 서비스 클라이언트에도 jwt token을 부여하도록 추후 고려가 되어야할 것 같아요..😳 그게 아니라면 내부의 각 라우트 경로들에 구분해서 별도로 들어가야할 것 같습니다!

Reply:
그러네요 논의가 필요한 부분이겠네요

Reply:
그럼 서비스 클라이언트 요청들은 프로젝트 라우터 내에서 컨트롤러 파일을 분리해서 관리해야겠네요!
verifyToken은 라우터 내 요청들마다 실행되게 해야할까요?

Reply:
```suggestion
router.use("/projects/:projectId/service/auth", serviceRoute);
router.use("/projects", verifyToken, projectRoute);
```
이런 식으로 해결 할 수도 있을 것 같네요

Reply:
오 이런 방법이 있었네요! 👍 👍